app/human-detector-app/screens/CameraScreen.tsx

```diff
@@ -56,31 +55,32 @@ const styles = StyleSheet.create({
 });
 
 type Props = NativeStackScreenProps<RootStackParamList, 'Cameras'>;
-export default function CameraScreen({ navigation }: Props): React.ReactElement {
-  const cameraOne: Camera = new Camera('123', "AAAAA's Camera", '99');
-  const cameraTwo: Camera = new Camera('124', "BBBBB's Camera", '725');
-  const cameraThree: Camera = new Camera('125', "CCCCC's Camera", '400');
+export default function CameraScreen({ navigation, route }: Props): React.ReactElement {
+  const userContext = React.useContext(UserContext);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const isFocused = useIsFocused();
+  const { groupId } = route.params;
 
-  const [listOfCameras] = useState([cameraOne, cameraTwo, cameraThree]);
+  if (!userContext) {
+    console.error('no user context in camera screen!');
+    throw new Error('no user context in camera screen');
+  }
+  const groupToView = userContext.getGroupFromId(groupId);
+  if (!groupToView) {
+    throw new Error('Group to view not valid');
+  }
```

Comment:
Why not just pass the group object directly and get the cameras from there? Maybe I'm just dumb